generators/entity-server/templates/src/main/java/package/domain/Entity.java.jhi.spring_data_couchbase.ejs

```diff
@@ -100,14 +99,17 @@ import java.util.stream.Collectors;
 
 <&_ } -&>
 <%_ } -%>
+<% Commented out until N1qlJoin support is fixed */%>
 
 <&_ if (fragment.classAdditionalRelationshipsSection) { -&>
     <%_ for (const relationship of relationships) { _%>
-      <%_ if (!relationship.id) { _%>
+      <%_ if (!relationship.id && !relationship.otherEntity.embedded) { _%>
         <%_ if (relationship.collection) { _%>
+    @JsonIgnore
     @Field("<%= relationship.relationshipFieldNamePlural %>")
```

Comment:
Since you reverted back to underscored field names, shouldn't we do the same for relationships also ?

Reply:
personally, I prefer to keep capitalized names everywhere, including `AbstractAuditingEntity`. WDYT @mshima, @gkysaad ?

Reply:
@tchlyah my opinion is that we should try to use the technology standard.
If it is camelCase, I am ok to it.
But we should preferably use:
```
  @Field("<%= field.columName %>")
```

Hardcoded:
```
<%= this.getColumnName('first_user') %>
```

And customize columName according to the database at `getColumnName()`,  at columName logic and probably others.
https://github.com/mshima/generator-jhipster/blob/de93ba28ada805fdcfaa4d20351c9bfbf7d20f16/utils/field.js#L293-L309

This approach:
- allow more flexibility to blueprints.
- allow to add prefix to reserved words.

Mixing snakeCase with camelBase is worst than only snakeCase.

Reply:
Couchbase is using Json, which convention is [camelCase](https://jsonapi.org/recommendations/). Also JHipster is using camelCase for REST APIs. So I see no point where we should save the document differently from what we send to clients.

I understand the interest of using ‘getColumnName()’ method, but I think it’ll be to hard to implement, specially for relationships. This is why those fields were using camelCase. Also, Spring Data Couchbase automatically escape fields using backticks, so we can safely use any reserved keywords.

I think it’s much simpler to use ‘@Field’ without any custom name. This way we can easily avoid mixing ‘snake_case’ with ‘camelCase’.

Reply:
+1 to the technology standard.

Reply:
I'll revert it back to using the `@Field` annotation with no custom name, in order to keep it as `camelCase`

Reply:
I've made the changes to change everything to `camelCase`.